testlint/testCerts/issuerDNDuplicateAttribute.cer

```diff
@@ -0,0 +1,3 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
PEM files do not look like this. They have 64 characters per line. Please fix all of these.